themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -254,7 +254,7 @@ includeInAccessToken.label=Add to access token
 includeInAccessToken.tooltip=Should the claim be added to the access token?
 includeInUserInfo.label=Add to userinfo
 includeInUserInfo.tooltip=Should the claim be added to the userinfo?
-usermodel.clientRoleMapping.clientId.label=Client ID
+usermodel.clientRoleMapping.clientId.label=Specific Client ID
```

Comment:
Please revert to Client ID to be consistent with the rest of the code. Apologies I didn't spot this earlier - this is the very last change :)

Reply:
Honestly, I'm not sure about that. I think this solution avoids some confusions about functionality as you can see in the JIRA ticket. IMO, the admin doesn't even read the tooltip when see the "Client ID" label. There is described purpose of the field in the tooltip which is different than normal Client ID. So, this should, at least, force the admins to read the tooltip and avoid confusions.

Reply:
There are many places across admin console where client role is chosen as a pair of _Client ID / role name_, not as _Specific Client ID / role name_. See e.g. authorization settings in clients.
I am afraid that no label would help without reading documentation and proper understanding of what is the purpose of individual fields. Adding `Specific` makes this field inconsistent with the rest of the UI.
Please revert.